paths/release_triggers/update.yaml

```
@@ -0,0 +1,92 @@
+---
+summary: Update a release trigger
+description: Update a recurring release.
+operationId: release_triggers/update
+tags:
+  - Release Triggers
+parameters:
+  - "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+  - "$ref": "../../parameters.yaml#/account_id"
+  - "$ref": "../../parameters.yaml#/distribution_id"
+  - "$ref": "../../parameters.yaml#/id"
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        title: release/update/parameters
+        properties:
+          cron_schedule:
+            description: Cron schedule for the scheduler. Read more about the format of this field at https://en.wikipedia.org/wiki/Cron
+            type: string
+            example: "15 18 1,3 * *"
+          time_zone:
+            description: Time zone for the scheduler
+            type: string
+            example: "Europe/Berlin"
+          locale_ids:
+            description: List of locale ids that will be included in the release. If empty, distribution locales will be used
```

I think we agreed for now to remove fallback locale_ids until we have some further clarity on it

```suggestion
            description: List of locale ids that will be included in the release.
```